setup.cfg

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+[metadata]
+name = invenio-rdm-migrator
+version = attr: invenio_rdm_migrator.__version__
+description = InvenioRDM module for data migration.
+long_description = file: README.rst, CHANGES.rst
+keywords = invenio rdm migration
+license = MIT
+author = CERN
+author_email = info@inveniosoftware.org
+platforms = any
+url = https://github.com/inveniosoftware/invenio-rdm-migrator
+classifiers =
+    Development Status :: 3 - Alpha
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.7
+zip_safe = False
+install_requires =
+    invenio-records[postgresql]>=2.0.1
```

Comment:
we need this due to `dict_set` which is completely non-invenio dependent. However, I do not think that is a good idea to start refactoring/moving that code since it is widely used in the codebase.